truss-chains/truss_chains/stub.py

```diff
@@ -97,7 +305,21 @@ def _client_cycle_needed(self, cached_client: Optional[tuple[Any, int]]) -> bool
             or (int(time.time()) - cached_client[1]) > self._client_cycle_time_sec
         )
 
-    def _client_sync(self) -> httpx.Client:
+    def _log_retry(self, retry_state: tenacity.RetryCallState) -> None:
+        attempt_number = retry_state.attempt_number
+        if attempt_number > 1:
```

Comment:
Wondering if `_log_retry` would only get called on fail since it's attached to `before_sleep`... should this condition be removed?

Reply:
Oh very good catch!